### PR TITLE
Migration to GitHub

### PR DESCRIPTION
**Pull request by joshmoore**
- [x] activate travis
 - [x] check that logo in README is green
 - [x] move gitlab repo to `graduated` and archive
 - [x] remove mirrored repository
 - [x] fix readme
 - [x] add new token
 - [x] allow publishing from travis

Then use bumpversion to release to pypi.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.. image:: https://travis-ci.org/openmicroscopy/omero-signup.svg?branch=master
-    :target: https://travis-ci.org/openmicroscopy/omero-webtest
+.. image:: https://travis-ci.org/ome/omero-signup.svg?branch=master
+    :target: https://travis-ci.org/ome/omero-webtest
```
Comment (sbesson) [inline README.rst]:
```suggestion
    :target: https://travis-ci.org/ome/omero-signup
```

**Review by sbesson — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-VERSION = '0.0.9'
+VERSION = '0.1.0'
```
Comment (manics) [inline setup.py]: I've previously used bumpversion for this: https://github.com/ome/omero-signup/tree/v0.0.9#release-process

If we move to a fully manual process https://github.com/ome/omero-signup/blob/v0.0.9/.bumpversion.cfg should be deleted and the README updated

Comment (joshmoore) [reply setup.py]: I'll revert and give it a try.

**Comment by joshmoore** [comment]
Moving on.

**Pull request merged**